sdk/storage/azure-storage-blob-batch/src/main/java/com/azure/storage/blob/batch/BlobBatchAsyncClient.java

```diff
@@ -48,7 +48,7 @@
 public final class BlobBatchAsyncClient {
     private static final ClientLogger LOGGER = new ClientLogger(BlobBatchAsyncClient.class);
 
-    private final AzureBlobStorageImpl client;
+    final AzureBlobStorageImpl client;
```

Comment:
Why are these no longer private?

Reply:
I needed to change this to package-private, though I'll update this to be accessed through a package-private getter, as Groovy/Spock did a lot of illegal reflective accesses and this was cleanup to have the JUnit tests abide by Java access permissions, ex can't access private properties of a type.